common/config/config.py

```diff
@@ -667,6 +668,10 @@ def const_gas_price(self) -> int | None:
             const_gas_price = min_gas_price
         return const_gas_price * (10**9)
 
+    @cached_property
+    def priority_fee_num_blocks_to_average(self) -> int:
```

Comment:
Please rename to:
`priority_fee_block_cnt_to_avg`

Reply:
done